README.md

```diff
@@ -392,22 +394,18 @@ A basic usage of this object template would involve the creation of `templates/u
 
 ```
 
-Please note that the UserAssignedIdentity name is set internally. This value is set in the `adp-flux-services` repository which follows standard naming conventions "{TEAM_MI_PREFIX}-{SERVICE_NAME}". For e.g. In SND1 if the `TEAM_MI_PREFIX` value is set to 'sndadpinfmid1401' and `SERVICE_NAME` value is set to 'adp-demo-service', then `managedIdName` value will be : sndadpinfmid1401-adp-demo-service .
+This template does not require you to supply any required values in the values.yaml in the file.
+
+This template uses the below values, whose values are set using platform variables in the `adp-flux-services` repository as a part of the service's ASO helmrelease value configuration, and you don't need to set them explicitly in the values.yaml file.
```

Comment:
I'd add in: Whilst the Platform orchestration will manage the 'platform' level variables, they can be optionally supplied in some circumstances. Examples include in sandpit/development when testing against team-specific infrastructure (that isn't Platform shared). So, if you have a dedicated Service Bus or Database server instance, you can point to those to ensure you apps works as expected. Otherwise, don't supply the Platform level variables as these will be automatically managed and orchestrated throughout all the environments appropriately against core shared infrastructure.

Reply:
@DaRoszko : I have added this paragraph in this section at the top as it is applicable to all the resources. 
Is that looks ok to you?
https://github.com/DEFRA/adp-aso-helm-library/blob/a9d486eae0b0df4c640629eb498aa392aacbe520/README.md#environment-specific-default-values